pages/blog/posts/custom-annotations-will-continue.md

```diff
@@ -13,7 +13,7 @@ authors:
 excerpt: "Custom annotations are one of JSON Schemas most-used features.  Here's how we're still supporting them."
 ---
 
-[Last time](./the-last-breaking-change), I wrote about how we had to remove support for unknown keywords.  I ended that post with a note saying that we were still searching for a way that we can support non-functional custom keywords, that is, simple annotations.
+[Last time](/blog/posts/the-last-breaking-change), I wrote about how we had to remove support for unknown keywords.  I ended that post with a note saying that we were still searching for a way that we can support non-functional custom keywords, that is, simple annotations.
```

Comment:
The url here and the rest of the links from one blog to another should be just the name of the page because all are in the same folder.